### PR TITLE
Bring back ics feed component

### LINKED ISSUE #7: Calendar ICS link throws an error
In the current release version, attempting to grab the ics link from the calendar throws the following error:

```
Compile Error: ics-feed is not a helper

Error: Compile Error: ics-feed is not a helper
    at t.compile (https://d26vzvixg52o0d.cloudfront.net/assets/vendor-7209fae8c902c71f1423222c3811bc3e.js:4573:82)
    at t.compile (https://d26vzvixg52o0d.cloudfront.net/assets/vendor-7209fae8c902c71f1423222c3811bc3e.js:4483:234)
    at t.e.compileStatement (https://d26vzvixg52o0d.cloudfront.net/assets/vendor-7209fae8c902c71f1423222c3811bc3e.js:4268:208)
    at t.compile (https://d26vzvixg52o0d.cloudfront.net/assets/vendor-7209fae8c902c71f1423222c3811bc3e.js:4280:6)
    at a.compile (https://d26vzvixg52o0d.cloudfront.net/assets/vendor-7209fae8c902c71f1423222c3811bc3e.js:3928:41)
    at e.invokeBlock (https://d26vzvixg52o0d.cloudfront.net/assets/vendor-7209fae8c902c71f1423222c3811bc3e.js:4654:201)
    at t.evaluate (https://d26vzvixg52o0d.cloudfront.net/assets/vendor-7209fae8c902c71f1423222c3811bc3e.js:4234:153)
    at e.execute (https://d26vzvixg52o0d.cloudfront.net/assets/vendor-7209fae8c902c71f1423222c3811bc3e.js:4653:112)
    at e.resume (https://d26vzvixg52o0d.cloudfront.net/assets/vendor-7209fae8c902c71f1423222c3811bc3e.js:4650:474)
    at t.handleException (https://d26vzvixg52o0d.cloudfront.net/assets/vendor-7209fae8c902c71f1423222c3811bc3e.js:4689:514) 
```

### PR DESCRIPTION
**Pull request by stopfstedt**
fixes #7 

this requires a new version of this add-on to be published to npm.

### CODE REVIEW

**Comment by jrjohnson** [comment]
Deploy preview ready!

Built with commit 23eb2c5fce7bfad474e428e5fc7ec65db34e33fe

https://deploy-preview-8--ilios-common.netlify.com

**Comment by jrjohnson** [comment]
Deploy preview ready!

Built with commit 1c98555eeb753082a706a9d53fed6c46c99105c6

https://deploy-preview-8--ilios-common.netlify.com

**Comment by stopfstedt** [comment]
selection_304

**Comment by saschaben** [comment]
LGTM 👍

File: addon/templates/components/dashboard-calendar.hbs [hunk 1]
```diff
@@ -169,5 +169,6 @@
     changeView=changeView
     selectEvent=selectEvent
     icsFeedUrl=(get (await currentUser.model) 'absoluteIcsUri')
+    icsInstructions=icsInstructionsTranslation
```
Comment (jrjohnson) [inline addon/templates/components/dashboard-calendar.hbs]: Not going to hold this up over this, but we should move away from this pattern and instead do:
`icsInstructions=(t 'icsInstructions')` to pass in the translated text or in this case just do the translation inside of the `ics-feed` component not that the translations exist in the same repo as the component.

**Pull request merged**